javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -2,22 +2,78 @@ import React from "react";
 import BootstrapTable from 'react-bootstrap-table-next';
 
 const BasicCourseTable = ( {classes} ) => {
-  
+
+  const sections = [];
+  classes.forEach(
+    (course) => {
+      course.classSections.forEach(
+        (section) => {
+          section.course =
+          {
+          courseId: course.courseId,
+          title: course.title,
+          unitsFixed: course.unitsFixed
+          };
+         sections.push(section);
+       }
+      )
+    }
+  );
+const rowStyle = (row, rowIndex) => {
+  console.log(`row=`,row, `rowIndex`, rowIndex);
+  return (row.section % 100 == 0)? {backgroundColor: '#CEDEFA'}: (rowIndex % 2 == 1)? {backgroundColor: '#EDF3FE'}: {backgroundColor: '#FFFFFF'}
+}
+const dataAlignment = (cell, row) => {
+  // console.log(`cell=${cell} row=`, row);
+  const alignmnet = (row.section % 100 == 0)? 'left': 'right';
+  return alignmnet
+}
+const renderCourseId = (cell, row) => {
+  // console.log(`cell=${cell} row=`, row);
```

Comment:
We probably don't need this anymore.

Reply:
Which line are you saying to remove for this specific PR?